### PR TITLE
Converted page to HTML (closes #7, #9)

### LINKED ISSUE #7: The character encoding of the HTML document not declared.
Need to add `<meta charset=utf-8>`

### PR DESCRIPTION
**Pull request by marcoscaceres**
Now valid HTML. Checked with validator.nu.

### CODE REVIEW

File: newTab.html [hunk 1]
```diff
@@ -1,143 +1,110 @@
...
-  </div>
-
-  <div id="newtab-scrollbox">
-
-    <div id="newtab-vertical-margin">
-
-      <div id="newtab-margin-top"></div>
-
-      <div id="newtab-margin-undo-container">
-        <div id="newtab-undo-container" undo-disabled="true">
```
Comment (emtwo) [inline newTab.html:86]: In undo.js we add and remove `undo-disabled` and have specific CSS for it so switching it to `data-undo-disabled` breaks some of the functionality

File: newTab.html [hunk 2]
```diff
@@ -1,143 +1,110 @@
...
+                <button onclick="gIntro._next()"></button>
+            </div>
         </div>
-        <div id="newtab-intro-buttons">
-          <input type="button" onclick="gIntro._back()"/>
-          <input type="button" default="true" onclick="gIntro._next()"/>
+        <div id="newtab-intro-footer">
+            <ul>
+                <li></li>
+                <li></li>
```
Comment (marcoscaceres) [inline newTab.html:63]: Any idea what these do?

File: newTab.html (hunk 1 above)
Comment (marcoscaceres) [inline newTab.html:86]: ok, will add to bug I filed over in other repo.

Comment (marcoscaceres) [inline newTab.html:86]: added to https://github.com/mozilla/remote-newtab/issues/3

File: newTab.html (hunk 2 above)
Comment (emtwo) [inline newTab.html:63]: They get populated with links in the code.

**Comment by emtwo** [comment]
I'm ok with merging this patch and keeping breakages as separate issues

**Pull request merged**